### PR TITLE
#1 Basic Spring Boot structure generated from http://start.spring.io/

### PR DESCRIPTION
**Pull request by cbeach47**
Provides a basic structure for the waffle menu service.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,22 @@
 # uPortal Waffle Menu Service
 
-A backing storage system to provide each user appropriate menu items.
+A RESTful service to provide each user appropriate menu items.
+
+Users are identified by JWT and the roles embedded in the JWT serves as a filter against the 'master' list of items.
+
+The master list of items are configured via a json file on the server of the form:
+
+```
```
Comment (ChristianMurphy) [inline README.md]:
```json

**Review by ChristianMurphy — APPROVED** [review]
Seems like a good start!
Thanks @cbeach47! :bow:

File: src/main/java/org/jasig/portlet/wafflemenu/controllers/rest/v1/WaffleMenuRestController.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ */
+@RestController
+@RequestMapping(WaffleMenuRestController.API_ROOT)
+public class WaffleMenuRestController {
+
+	public static final String API_ROOT = "/api/v1/waffle";
+
+	
+	@RequestMapping(method = RequestMethod.GET)
+   	public String fetchWaffleMenuContent() {
```
Comment (drewwills) [inline src/main/java/org/jasig/portlet/wafflemenu/controllers/rest/v1/WaffleMenuRestController.java:41]: AFAIK you will probably want to return a `List` of some sort here.

Comment (cbeach47) [reply src/main/java/org/jasig/portlet/wafflemenu/controllers/rest/v1/WaffleMenuRestController.java:41]: Yes, in the next iteration I'll be changing that.

File: README.md (hunk 1 above)
Comment (cbeach47) [reply README.md]: Thanks - fixed now.

**Pull request merged**